flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateChunkReaderTest.java

```diff
@@ -68,27 +81,28 @@ public void testBufferRecycledOnSuccess() throws IOException, InterruptedExcepti
                     .readChunk(stream, serializer.getHeaderLength(), handler, "channelInfo", 0);
         } finally {
             checkState(!handler.requestedBuffers.isEmpty());
-            assertTrue(
-                    handler.requestedBuffers.stream()
-                            .allMatch(TestChannelStateByteBuffer::isRecycled));
+            assertThat(
+                            handler.requestedBuffers.stream()
+                                    .allMatch(TestChannelStateByteBuffer::isRecycled))
+                    .isTrue();
         }
     }
 
     @Test
-    public void testBuffersNotRequestedForEmptyStream() throws IOException, InterruptedException {
+    void testBuffersNotRequestedForEmptyStream() throws IOException, InterruptedException {
         ChannelStateSerializer serializer = new ChannelStateSerializerImpl();
         TestRecoveredChannelStateHandler handler = new TestRecoveredChannelStateHandler();
 
         try (FSDataInputStream stream = getStream(serializer, 0)) {
             new ChannelStateChunkReader(serializer)
                     .readChunk(stream, serializer.getHeaderLength(), handler, "channelInfo", 0);
         } finally {
-            assertTrue(handler.requestedBuffers.isEmpty());
+            assertThat(handler.requestedBuffers.isEmpty()).isTrue();
```

Comment:
```suggestion
            assertThat(handler.requestedBuffers).isEmpty();
```
There is `isEmpty()` method from assertJ